frontend/src/routes/Governance/overview/PolicyViolationSummary.tsx

```diff
@@ -96,11 +96,7 @@ export function ViolationsCard(props: {
                         link: props.pending > 0 ? `${NavigationPath.policies}?violations=pending` : undefined,
                     },
                 ]}
-                colorScale={[
-                    'var(--pf-global--danger-color--100)',
-                    'var(--pf-global--success-color--100)',
-                    'var(--pf-global--warning-color--100)',
-                ]}
+                colorScale={['var(--pf-global--danger-color--100)', '#BBB', 'var(--pf-global--warning-color--100)']}
```

Comment:
A note on #BBB: This CSS color class corresponds to "--pf-chart-color-black-200", however, the "pf" version of this class does not seem to be one of the built in classes for the donut chart component, as using the key renders a solid black color. So I think I am forced to use the CSS class directly.